src/main/java/uk/gov/hmcts/reform/feature/security/Roles.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.feature.security;
+
+public final class Roles {
+
+    public static final String ROLE_USER = "USER";
+
+    public static final String ROLE_EDITOR = "EDITOR";
+
+    public static final String ROLE_ADMIN = "ADMIN";
```

Comment:
imho `ROLE_` prefix can now be removed when it's in a `Roles` class

Reply:
also, did you consider using enum?

Reply:
The more I have come up with cases like that - the more I see that enum is not actually beneficial. Unless you do something switch/case-y or other custom domain design. I wouldn't vote as a must have in here